js/ai/src/model.ts

```diff
@@ -145,6 +146,8 @@ export const ModelInfoSchema = z.object({
       context: z.boolean().optional(),
     })
     .optional(),
+  /** Mark the model as deprecated. */
+  deprecated: z.boolean().optional(),
```

Comment:
Could you please run this through discord?
I believe at some point we discussed that it might be useful to have a `status` fields with enum values along the lines of `['active', 'featured', 'legacy', 'deprecated']`

Reply:
```suggestion
  stage: z.enum(['featured', 'unstable', 'stable', 'legacy', 'deprecated']).optional(),
```